examples/somersaultecu.py

```diff
@@ -2445,7 +2518,210 @@ class SomersaultSID(IntEnum):
     negative_responses=NamedItemList(),
     global_negative_responses=NamedItemList(),
     import_refs=[],
-    state_charts=NamedItemList(),
+    state_charts=NamedItemList([
+        StateChart(
+            semantic="annoyed",
+            odx_id=OdxLinkId("charts.annoyed.chart", doc_frags),
+            oid=None,
+            short_name="annoyed_chart",
+            long_name=None,
+            description=None,  #Description("State chart for a day where the ECU is grumpy",
```

Comment:
be aware that this is due to some weirdness: if a description is added here, the resulting ODX-D file does not validate anymore although I'm 100% sure that the XSD includes an ELEMENT-ID group (and thus DESC) for STATE-CHART...

Reply:
Could be the order of elements in the XML

Reply:
I don't think so: The ELEMENT-ID subtags are written by the [`printElementIdSubtags()`](https://github.com/mercedes-benz/odxtools/blob/main/odxtools/templates/macros/printElementId.xml.jinja2#L13-L19) macro and it works fine for everything else. the error produced by xmllint also does not look like this is the case:
```
> xmllint --schema BaseStandard/Schema/odx.xsd --noout somersault.odx-d
somersault.odx-d:1220: Schemas validity error : Element 'DESC': Character content other than whitespace is not allowed because the content type is 'element-only'.
somersault.odx-d fails to validate
```

Reply:
You need to wrap it in `<p></p>`

Reply:
ah, I completely forgot. thanks the info :)